sys-fs/tmsu/tmsu-0.6.1.ebuild

```diff
@@ -10,28 +10,28 @@ DESCRIPTION="Files tagger and virtual tag-based filesystem"
 HOMEPAGE="https://github.com/oniony/TMSU/wiki"
 SRC_URI="https://github.com/oniony/TMSU/archive/v${PV}.tar.gz -> ${P}.tar.gz"
 
-LICENSE="GPL-3"
+LICENSE="GPL-3+"
 SLOT="0"
 KEYWORDS="~amd64"
 IUSE="test"
 
-RDEPEND="
-	dev-go/go-sqlite3
-	dev-libs/go-fuse
-	"
+RDEPEND=""
 DEPEND="
+	dev-go/go-sqlite3
 	dev-lang/go
-	test? ( ${RDEPEND} )
-	"
-
-S=${WORKDIR}/TMSU-${PV}
-
-src_compile() {
-	GOPATH="${S}:$(get_golibdir_gopath)" \
-		go install -v -work -x ${EGO_BUILD_FLAGS} ${EGO_PN} || die
+	dev-libs/go-fuse
+"
+
+src_unpack() {
+	if [ "${A}" != "" ]; then
```

Comment:
Instead of manually running `unpack` yourself, you can call `default` so that the default unpack phase takes place, then you can perform `mv`